lib/orderbook/TradingPair.ts

```diff
@@ -161,7 +161,20 @@ class TradingPair {
    * @returns the removed order or order portion, otherwise undefined if the order wasn't found
    */
   public removePeerOrder = (orderId: string, peerPubKey: string, quantityToRemove?: number): { order: PeerOrder, fullyRemoved: boolean} => {
```

Comment:
i'd say to keep `peerPubKey` optional both here and in `OrderBook.removePeerOrder`, same as we're doing with the quantity. Otherwise it's implicit.